diff --git a/Cargo.toml b/Cargo.toml
index 1b0cccc..d550c6b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -838,7 +838,7 @@ features = ["rt"]
 version = "0.3.0"

 [target.'cfg(all(tokio_unstable, target_has_atomic = "64"))'.dev-dependencies.tracing-mock]
-version = "= 0.1.0-beta.1"
+version = "= 0.1.0"

 [target.'cfg(all(tokio_unstable, target_os = "linux"))'.dependencies.backtrace]
 version = "0.3.58"
