diff --git a/Cargo.toml b/Cargo.toml
index c287b68..57adf82 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -85,10 +85,10 @@ exactly = 1
 [features]
 auto-color = [
     "color",
-    "anstream/auto",
+#    "anstream/auto",
 ]
 color = [
-    "dep:anstream",
+#    "dep:anstream",
     "dep:anstyle",
 ]
 default = [
@@ -133,11 +133,11 @@ path = "examples/in_tests.rs"
 name = "syslog_friendly_format"
 path = "examples/syslog_friendly_format.rs"

-[dependencies.anstream]
-version = "1.0.0"
-features = ["wincon"]
-optional = true
-default-features = false
+#[dependencies.anstream]
+#version = "1.0.0"
+#features = ["wincon"]
+#optional = true
+#default-features = false

 [dependencies.anstyle]
 version = "1.0.13"
