Index: webauthn-rs-core/Cargo.toml
===================================================================
--- webauthn-rs-core.orig/Cargo.toml
+++ webauthn-rs-core/Cargo.toml
@@ -58,7 +58,7 @@ version = "=0.5.1"
 version = "0.4.2"
 
 [dependencies.der-parser]
-version = "9.0"
+version = "10.0"
 
 [dependencies.hex]
 version = "0.4.3"
@@ -106,7 +106,7 @@ version = "=0.5.1"
 version = "=0.5.1"
 
 [dependencies.x509-parser]
-version = "0.16"
+version = "0.17"
 
 [dev-dependencies.hex-literal]
 version = "0.4"
