diff --git a/Cargo.toml b/Cargo.toml
index f095edd..96f7776 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -85,7 +85,7 @@ path = "tests/smoke.rs"
 name = "tests"
 path = "tests/tests.rs"

-[dependencies.ascii]
+[disabled.dependencies.ascii]
 version = "1.1"
 optional = true

@@ -93,7 +93,7 @@ optional = true
 version = "~1.6.1"
 optional = true

-[dependencies.bson]
+[disabled.dependencies.bson]
 version = "2"
 optional = true

