diff --git a/Cargo.toml b/Cargo.toml
index 0a5434e..f7d1a49 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -77,11 +77,11 @@ default-features = false
 version = "1.0"
 optional = true
 
-[dependencies.rkyv]
-version = "0.7.42"
-features = ["alloc"]
-optional = true
-default-features = false
+#[dependencies.rkyv]
+#version = "0.7.42"
+#features = ["alloc"]
+#optional = true
+#default-features = false
 
 [dependencies.serde]
 version = "1.0.25"
