diff --git a/Cargo.toml b/Cargo.toml
index 5220f8c..7cc56e7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -123,21 +123,6 @@ std = [
     "num-rational/std",
     "approx/std",
     "simba/std",
-    "glam014?/std",
-    "glam015?/std",
-    "glam016?/std",
-    "glam017?/std",
-    "glam018?/std",
-    "glam019?/std",
-    "glam020?/std",
-    "glam021?/std",
-    "glam022?/std",
-    "glam023?/std",
-    "glam024?/std",
-    "glam025?/std",
-    "glam027?/std",
-    "glam028?/std",
-    "glam029?/std",
     "glam030?/std",
 ]

