Index: walrus/Cargo.toml
===================================================================
--- walrus.orig/Cargo.toml
+++ walrus/Cargo.toml
@@ -55,7 +55,7 @@ harness = false
 version = "1.0"
 
 [dependencies.gimli]
-version = "0.26.0"
+version = "0.31.0"
 
 [dependencies.id-arena]
 version = "2.2.1"
@@ -74,10 +74,10 @@ optional = true
 version = "=0.22.0"
 
 [dependencies.wasm-encoder]
-version = "0.214.0"
+version = "0.217.0"
 
 [dependencies.wasmparser]
-version = "0.214.0"
+version = "0.217.0"
 
 [dev-dependencies.criterion]
 version = "0.5.0"
Index: walrus/src/module/mod.rs
===================================================================
--- walrus.orig/src/module/mod.rs
+++ walrus/src/module/mod.rs
@@ -239,13 +239,13 @@ impl Module {
                     let result =
                         match s.name() {
                             "producers" => wasmparser::ProducersSectionReader::new(
-                                BinaryReader::new(s.data(), s.data_offset(), wasm_features),
+                                BinaryReader::new_features(s.data(), s.data_offset(), wasm_features),
                             )
                             .map_err(anyhow::Error::from)
                             .and_then(|s| ret.parse_producers_section(s)),
                             "name" => {
                                 let name_section_reader = wasmparser::NameSectionReader::new(
-                                    BinaryReader::new(s.data(), s.data_offset(), wasm_features),
+                                    BinaryReader::new_features(s.data(), s.data_offset(), wasm_features),
                                 );
                                 ret.parse_name_section(name_section_reader, &indices)
                             }
Index: walrus/src/module/debug/dwarf.rs
===================================================================
--- walrus.orig/src/module/debug/dwarf.rs
+++ walrus/src/module/debug/dwarf.rs
@@ -133,6 +133,7 @@ where
                         timestamp: comp_file.timestamp(),
                         size: comp_file.size(),
                         md5: *comp_file.md5(),
+                        source: None,
                     }),
                 )
             }
@@ -182,6 +183,7 @@ where
                 timestamp: from_file.timestamp(),
                 size: from_file.size(),
                 md5: *from_file.md5(),
+                source: None,
             });
             files.push(program.add_file(from_name, from_dir, from_info));
         }
@@ -216,13 +218,13 @@ where
                     }
                     from_base_address = val;
 
-                    from_row.execute(read::LineInstruction::SetAddress(0), &mut from_program);
+                    from_row.execute(read::LineInstruction::SetAddress(0), &mut from_program)?;
                 }
                 read::LineInstruction::DefineFile(_) => {
                     return Err(write::ConvertError::UnsupportedLineInstruction);
                 }
                 _ => {
-                    if from_row.execute(instruction, &mut from_program) {
+                    if from_row.execute(instruction, &mut from_program)? {
                         if !program.in_sequence() {
                             // begin new sequence if exists
                             current_sequence_base_address = (self.convert_address)(
