diff --git a/Cargo.toml b/Cargo.toml
index 0218bbd..99a189f 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,8 +40,6 @@ repository = "https://github.com/rust-num/num-bigint"
 features = [
     "std",
     "serde",
-    "rand",
-    "quickcheck",
     "arbitrary",
 ]
 rustdoc-args = [
@@ -80,12 +78,12 @@ version = "0.2.18"
 features = ["i128"]
 default-features = false

-[dependencies.quickcheck]
+[disabled.dependencies.quickcheck]
 version = "1"
 optional = true
 default-features = false

-[dependencies.rand]
+[disabled.dependencies.rand]
 version = "0.8"
 optional = true
 default-features = false
@@ -98,8 +96,8 @@ default-features = false
 [features]
 arbitrary = ["dep:arbitrary"]
 default = ["std"]
-quickcheck = ["dep:quickcheck"]
-rand = ["dep:rand"]
+quickcheck = []
+rand = []
 serde = ["dep:serde"]
 std = [
     "num-integer/std",
