diff --git a/build.rs b/build.rs
index 28d0d47..764d6c2 100644
--- a/build.rs
+++ b/build.rs
@@ -60,6 +60,8 @@ fn main() {
     use std::env::var;
+     use std::env;
 
     println!("cargo:rustc-check-cfg=cfg(switchable_stack,asm,link_asm)");
+    println!("dh-cargo:deb-built-using=psm_s=1={}", env::var("CARGO_MANIFEST_DIR").unwrap());
 
     if var("CARGO_CFG_MIRI").is_ok() {
         // Miri doesn't have a stack limit and the inline asm wouldn't work on miri anyway.
