diff --git a/Cargo.toml b/Cargo.toml
index 0c775a6..9b66569 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,29 +27,29 @@ repository = "https://github.com/ramosbugs/oauth2-rs"
 [package.metadata.docs.rs]
 all-features = true
 
-[[example]]
-name = "github"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "github"
+#required-features = ["reqwest-blocking"]
 
-[[example]]
-name = "google"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "google"
+#required-features = ["reqwest-blocking"]
 
-[[example]]
-name = "google_devicecode"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "google_devicecode"
+#required-features = ["reqwest-blocking"]
 
-[[example]]
-name = "letterboxd"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "letterboxd"
+#required-features = ["reqwest-blocking"]
 
-[[example]]
-name = "msgraph"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "msgraph"
+#required-features = ["reqwest-blocking"]
 
-[[example]]
-name = "wunderlist"
-required-features = ["reqwest-blocking"]
+#[[example]]
+#name = "wunderlist"
+#required-features = ["reqwest-blocking"]
 
 [dependencies.base64]
 version = ">= 0.21, <0.23"
@@ -70,10 +70,10 @@ version = "1.0"
 [dependencies.rand]
 version = "0.8"
 
-[dependencies.reqwest]
-version = "0.12"
-optional = true
-default-features = false
+#[dependencies.reqwest]
+#version = "0.12"
+#optional = true
+#default-features = false
 
 [dependencies.serde]
 version = "1.0"
@@ -121,13 +121,13 @@ features = ["v4"]
 
 [features]
 default = [
-    "reqwest",
-    "rustls-tls",
+#    "reqwest",
+#    "rustls-tls",
 ]
-native-tls = ["reqwest/native-tls"]
+#native-tls = ["reqwest/native-tls"]
 pkce-plain = []
-reqwest-blocking = ["reqwest/blocking"]
-rustls-tls = ["reqwest/rustls-tls"]
+#reqwest-blocking = ["reqwest/blocking"]
+#rustls-tls = ["reqwest/rustls-tls"]
 timing-resistant-secret-traits = []
 
 [target."cfg(not(target_arch = \"wasm32\"))".dependencies.curl]
