diff --git a/Cargo.toml b/Cargo.toml
index 68a57cb..f4bde8a 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@

 [package]
 edition = "2021"
-rust-version = "1.88"
+#rust-version = "1.85"
 name = "serde_with_macros"
 version = "3.18.0"
 authors = ["Jonas Bushart"]
@@ -114,7 +114,7 @@ name = "version_numbers"
 path = "tests/version_numbers.rs"

 [dependencies.darling]
-version = "0.23.0"
+version = "0.20.0"

 [dependencies.proc-macro2]
 version = "1.0.1"
