diff --git a/Cargo.toml b/Cargo.toml
index f445e18..dbaf579 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -177,7 +177,7 @@ std = ["alloc"]
 test_logging = []
 unstable-testing-arm-no-hw = []
 unstable-testing-arm-no-neon = []
-wasm32_unknown_unknown_js = ["getrandom/js"]
+wasm32_unknown_unknown_js = ["getrandom/js"]

 [lib]
 name = "ring"
@@ -244,7 +244,7 @@ version = "1.0.0"
 default-features = false

 [dependencies.getrandom]
-version = "0.2.10"
+version = "0.2.10"

 [dependencies.untrusted]
 version = "0.9"
@@ -253,12 +253,12 @@ version = "0.9"
 version = "1.2.8"
 default-features = false

-[target.'cfg(all(all(target_arch = "aarch64", target_endian = "little"), target_os = "windows"))'.dependencies.windows-sys]
-version = "0.52"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Threading",
-]
+#[target.'cfg(all(all(target_arch = "aarch64", target_endian = "little"), target_os = "windows"))'.dependencies.windows-sys]
+#version = "0.52"
+#features = [
+#    "Win32_Foundation",
+#    "Win32_System_Threading",
+#]

 [target.'cfg(all(all(target_arch = "aarch64", target_endian = "little"), target_vendor = "apple", any(target_os = "ios", target_os = "macos", target_os = "tvos", target_os = "visionos", target_os = "watchos")))'.dependencies.libc]
 version = "0.2.155"
@@ -268,10 +268,10 @@ default-features = false
 version = "0.2.148"
 default-features = false

-[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dev-dependencies.wasm-bindgen-test]
-version = "0.3.37"
-features = ["std"]
-default-features = false
+#[target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dev-dependencies.wasm-bindgen-test]
+#version = "0.3.37"
+#features = ["std"]
+#default-features = false

 [target.'cfg(any(unix, windows, target_os = "wasi"))'.dev-dependencies.libc]
 version = "0.2.148"
