diff --git a/Cargo.toml b/Cargo.toml
index 738d7da..06a3526 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,31 +38,31 @@ repository = "https://github.com/rust-openssl/rust-openssl"
 openssl = "1.0.1"

 [features]
-aws-lc = ["dep:aws-lc-sys"]
-aws-lc-fips = ["dep:aws-lc-fips-sys"]
-unstable_boringssl = ["bssl-sys"]
-vendored = ["openssl-src"]
+aws-lc = ["dep:aws-lc-sys"]
+aws-lc-fips = ["dep:aws-lc-fips-sys"]
+unstable_boringssl = ["bssl-sys"]
+vendored = ["openssl-src"]

 [lib]
 name = "openssl_sys"
 path = "src/lib.rs"

-[dependencies.aws-lc-fips-sys]
-version = "0.13"
-features = [
-    "ssl",
-    "bindgen",
-]
-optional = true
+[dependencies.aws-lc-fips-sys]
+version = "0.13"
+features = [
+    "ssl",
+    "bindgen",
+]
+optional = true

-[dependencies.aws-lc-sys]
-version = "0.38"
-features = ["ssl"]
-optional = true
+[dependencies.aws-lc-sys]
+version = "0.38"
+features = ["ssl"]
+optional = true

-[dependencies.bssl-sys]
-version = "0.1.0"
-optional = true
+[dependencies.bssl-sys]
+version = "0.1.0"
+optional = true

 [dependencies.libc]
 version = "0.2"
@@ -75,10 +75,10 @@ optional = true
 [build-dependencies.cc]
 version = "1.0.61"

-[build-dependencies.openssl-src]
-version = "300.2.0"
-features = ["legacy"]
-optional = true
+[build-dependencies.openssl-src]
+version = "300.5"
+features = ["legacy"]
+optional = true

 [build-dependencies.pkg-config]
 version = "0.3.9"
