diff --git a/Cargo.toml b/Cargo.toml
index 1baa4c1..ca1ceb5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -69,6 +69,7 @@ brotli = [
 custom-provider = []
 default = [
     "aws_lc_rs",
+    "ring",
     "logging",
     "prefer-post-quantum",
     "std",
@@ -95,7 +96,7 @@ std = [
     "once_cell/std",
 ]
 tls12 = []
-zlib = ["dep:zlib-rs"]
+#zlib = ["dep:zlib-rs"]
 
 [lib]
 name = "rustls"
@@ -126,14 +127,14 @@ default-features = false
 version = "5.0.0"
 optional = true
 
-[dependencies.hashbrown]
-version = "0.15"
-features = [
-    "default-hasher",
-    "inline-more",
-]
-optional = true
-default-features = false
+#[dependencies.hashbrown]
+#version = "0.15"
+#features = [
+#    "default-hasher",
+#    "inline-more",
+#]
+#optional = true
+#default-features = false
 
 [dependencies.log]
 version = "0.4.8"
@@ -169,9 +170,9 @@ package = "rustls-webpki"
 [dependencies.zeroize]
 version = "1.7"
 
-[dependencies.zlib-rs]
-version = "0.5"
-optional = true
+#[dependencies.zlib-rs]
+#version = "0.5"
+#optional = true
 
 [dev-dependencies.base64]
 version = "0.22"
