Index: wasm-bindgen-wasm-conventions/Cargo.toml
===================================================================
--- wasm-bindgen-wasm-conventions.orig/Cargo.toml
+++ wasm-bindgen-wasm-conventions/Cargo.toml
@@ -49,7 +49,7 @@ version = "0.4"
 version = "0.23"
 
 [dependencies.wasmparser]
-version = "0.212"
+version = "0.217"
 
 [lints.clippy]
 large_enum_variant = "allow"
Index: wasm-bindgen-wasm-conventions/src/lib.rs
===================================================================
--- wasm-bindgen-wasm-conventions.orig/src/lib.rs
+++ wasm-bindgen-wasm-conventions/src/lib.rs
@@ -192,7 +192,7 @@ pub fn target_feature(module: &Module, f
             .as_any()
             .downcast_ref()
             .context("failed to read section")?;
-        let mut reader = BinaryReader::new(&section.data, 0, WasmFeatures::default());
+        let mut reader = BinaryReader::new_features(&section.data, 0, WasmFeatures::default());
         // The first integer contains the target feature count.
         let count = reader.read_var_u32()?;
 
@@ -237,7 +237,7 @@ pub fn insert_target_feature(module: &mu
             .as_any_mut()
             .downcast_mut()
             .context("failed to read section")?;
-        let mut reader = BinaryReader::new(&section.data, 0, WasmFeatures::default());
+        let mut reader = BinaryReader::new_features(&section.data, 0, WasmFeatures::default());
         // The first integer contains the target feature count.
         let count = reader.read_var_u32()?;
 
