diff --git a/Cargo.toml b/Cargo.toml
index 77fc096..bc58dfa 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -42,9 +42,9 @@ rustdoc-args = [
 [dependencies.cfg-if]
 version = "1.0.0"

-[dependencies.core_arch]
-version = "0.1.5"
-optional = true
+#[dependencies.core_arch]
+#version = "0.1.5"
+#optional = true

 [dependencies.num-traits]
 version = "0.2.14"
@@ -63,9 +63,9 @@ default = []
 into_bits = []
 libcore_neon = []

-[target."cfg(target_arch = \"x86_64\")".dependencies.sleef-sys]
-version = "0.1.2"
-optional = true
+#[target."cfg(target_arch = \"x86_64\")".dependencies.sleef-sys]
+#version = "0.1.2"
+#optional = true

 [target.wasm32-unknown-unknown.dev-dependencies.wasm-bindgen]
 version = "=0.2.87"
