diff --git a/Cargo.toml b/Cargo.toml
index 49eebad..449643a 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -36,7 +36,6 @@ keywords = [
     "terminal",
     "color",
     "strip",
-    "wincon",
 ]
 categories = ["command-line-interface"]
 license = "MIT OR Apache-2.0"
@@ -89,10 +88,8 @@ exactly = 1
 auto = ["dep:anstyle-query"]
 default = [
     "auto",
-    "wincon",
 ]
 test = []
-wincon = ["dep:anstyle-wincon"]

 [lib]
 name = "anstream"
@@ -142,10 +139,6 @@ version = "1.7.0"
 [dev-dependencies.strip-ansi-escapes]
 version = "0.2.1"

-[target."cfg(windows)".dependencies.anstyle-wincon]
-version = "3.0.5"
-optional = true
-
 [lints.clippy]
 bool_assert_comparison = "allow"
 branches_sharing_code = "allow"
