diff --git a/Cargo.toml b/Cargo.toml
index 88ae42e..9ff9617 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -55,7 +55,7 @@ features = [

 [features]
 __test_doc_lints = []
-__ui_tests = ["trybuild"]
+__ui_tests = []
 adt_const_marker = ["rust_stable"]
 alloc = []
 const_marker = []
@@ -74,7 +74,7 @@ rust_stable = ["rust_1_83"]
 name = "typewit"
 path = "src/lib.rs"

-[dependencies.trybuild]
+[disabled.dependencies.trybuild]
 version = "1.0"
 optional = true

