diff --git a/Cargo.toml b/Cargo.toml
index 54b890e..a587919 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,7 +35,7 @@ repository = "https://github.com/sunng87/handlebars-rust"
 features = [
     "dir_source",
     "script_helper",
-    "rust-embed",
+#    "rust-embed",
 ]
 rustdoc-args = [
     "--cfg",
@@ -79,10 +79,10 @@ features = [
 ]
 optional = true

-[dependencies.rust-embed]
-version = "8.0.0"
-features = ["include-exclude"]
-optional = true
+#[dependencies.rust-embed]
+#version = "8.0.0"
+#features = ["include-exclude"]
+#optional = true

 [dependencies.serde]
 version = "1.0.0"
@@ -93,9 +93,9 @@ version = "1.0.39"
 [dependencies.thiserror]
 version = "1"

-[dependencies.walkdir]
-version = "2.2.3"
-optional = true
+[dependencies.walkdir]
+version = "2.5"
+optional = true

 [dev-dependencies.criterion]
 version = "0.5"
@@ -122,7 +122,7 @@ version = "0.12"

 [features]
 default = []
-dir_source = ["walkdir"]
+dir_source = ["walkdir"]
 no_logging = []
 script_helper = ["rhai"]
 string_helpers = ["heck"]
