diff --git a/Cargo.toml b/Cargo.toml
index 1bd2671..d87d3aa 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -55,7 +55,7 @@ path = "src/lib.rs"
 [dependencies.const_format_proc_macros]
 version = "=0.2.34"

-[dependencies.konst]
+[disabled.dependencies.konst]
 version = "0.2.13"
 optional = true
 default-features = false
@@ -102,7 +102,5 @@ nightly_const_generics = ["const_generics"]
 rust_1_51 = []
 rust_1_64 = [
     "rust_1_51",
-    "konst",
-    "konst/rust_1_64",
 ]
 rust_1_83 = ["rust_1_64"]
