diff --git a/Cargo.toml b/Cargo.toml
index d7cb779..a5d85f2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -68,14 +68,14 @@ async-std1 = [
 ]
 async-std1-rustls = [
     "async-std1",
-    "rustls",
+#    "rustls",
     "dep:futures-rustls",
 ]
 async-std1-rustls-tls = [
     "async-std1-rustls",
     "rustls-tls",
 ]
-aws-lc-rs = ["rustls?/aws-lc-rs"]
+#aws-lc-rs = ["rustls?/aws-lc-rs"]
 boring-tls = ["dep:boring"]
 builder = [
     "dep:httpdate",
@@ -108,18 +108,18 @@ file-transport-envelope = [
     "file-transport",
 ]
 fips = [
-    "aws-lc-rs",
-    "rustls?/fips",
+#    "aws-lc-rs",
+#    "rustls?/fips",
 ]
 mime03 = ["dep:mime"]
 pool = ["dep:futures-util"]
-ring = ["rustls?/ring"]
-rustls = ["dep:rustls"]
-rustls-no-provider = ["rustls"]
+#ring = ["rustls?/ring"]
+#rustls = ["dep:rustls"]
+#rustls-no-provider = ["rustls"]
 rustls-tls = [
-    "webpki-roots",
-    "rustls",
-    "ring",
+#    "webpki-roots",
+#    "rustls",
+#    "ring",
 ]
 sendmail-transport = [
     "tokio1_crate?/process",
@@ -153,15 +153,15 @@ tokio1-native-tls = [
 ]
 tokio1-rustls = [
     "tokio1",
-    "rustls",
-    "dep:tokio1_rustls",
+#    "rustls",
+#    "dep:tokio1_rustls",
 ]
 tokio1-rustls-tls = [
     "tokio1-rustls",
     "rustls-tls",
 ]
 web = ["dep:web-time"]
-webpki-roots = ["dep:webpki-roots"]
+#webpki-roots = ["dep:webpki-roots"]
 
 [lib]
 name = "lettre"
@@ -381,23 +381,23 @@ optional = true
 version = "0.9"
 optional = true
 
-[dependencies.rustls]
-version = "0.23.18"
-features = [
-    "logging",
-    "std",
-    "tls12",
-]
-optional = true
-default-features = false
+#[dependencies.rustls]
+#version = "0.23.18"
+#features = [
+#    "logging",
+#    "std",
+#    "tls12",
+#]
+#optional = true
+#default-features = false
 
-[dependencies.rustls-native-certs]
-version = "0.8"
-optional = true
+#[dependencies.rustls-native-certs]
+#version = "0.8"
+#optional = true
 
-[dependencies.rustls-platform-verifier]
-version = "0.6.0"
-optional = true
+#[dependencies.rustls-platform-verifier]
+#version = "0.6.0"
+#optional = true
 
 [dependencies.serde]
 version = "1.0.110"
@@ -432,15 +432,15 @@ version = "0.3"
 optional = true
 package = "tokio-native-tls"
 
-[dependencies.tokio1_rustls]
-version = "0.26"
-features = [
-    "logging",
-    "tls12",
-]
-optional = true
-default-features = false
-package = "tokio-rustls"
+#[dependencies.tokio1_rustls]
+#version = "0.26"
+#features = [
+#    "logging",
+#    "tls12",
+#]
+#optional = true
+#default-features = false
+#package = "tokio-rustls"
 
 [dependencies.tracing]
 version = "0.1.16"
@@ -457,9 +457,9 @@ version = "1"
 features = ["v4"]
 optional = true
 
-[dependencies.webpki-roots]
-version = "1.0.0"
-optional = true
+#[dependencies.webpki-roots]
+#version = "1.0.0"
+#optional = true
 
 [dev-dependencies.async-std]
 version = "1.8"
