diff --git a/Cargo.toml b/Cargo.toml
index 7394491..33cead1 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -86,8 +86,8 @@ version = "0.1.5"
 [target.'cfg(target_os = "haiku")'.dependencies.iana-time-zone-haiku]
 version = "0.1.1"

-[target.'cfg(target_os = "windows")'.dependencies.windows-core]
-version = ">=0.56, <=0.62"
+#[target.'cfg(target_os = "windows")'.dependencies.windows-core]
+#version = ">=0.56, <=0.62"

 [target.'cfg(target_vendor = "apple")'.dependencies.core-foundation-sys]
 version = "0.8.6"
