diff --git a/Cargo.toml b/Cargo.toml
index fef42c4..c2b56dd 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -60,7 +60,7 @@ targets = [
     "i686-linux-android",
     "x86_64-apple-darwin",
     "x86_64-pc-solaris",
-    "x86_64-pc-windows-msvc",
+#    "x86_64-pc-windows-msvc",
     "x86_64-unknown-freebsd",
     "x86_64-unknown-fuchsia",
     "x86_64-unknown-illumos",
@@ -79,8 +79,8 @@ allowed_external_types = [
     "libc::*::socklen_t",
     "libc::sa_family_t",
     "libc::*::sa_family_t",
-    "windows_sys::Win32::Networking::WinSock::socklen_t",
-    "windows_sys::Win32::Networking::WinSock::ADDRESS_FAMILY",
+#    "windows_sys::Win32::Networking::WinSock::socklen_t",
+#    "windows_sys::Win32::Networking::WinSock::ADDRESS_FAMILY",
 ]

 [features]
@@ -93,12 +93,12 @@ path = "src/lib.rs"
 [target.'cfg(any(unix, target_os = "wasi"))'.dependencies.libc]
 version = "0.2.172"

-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.60, <0.62"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_System_IO",
-    "Win32_System_Threading",
-    "Win32_System_WindowsProgramming",
-]
+#[target."cfg(windows)".dependencies.windows-sys]
+#version = ">=0.60, <0.62"
+#features = [
+#    "Win32_Foundation",
+#    "Win32_Networking_WinSock",
+#    "Win32_System_IO",
+#    "Win32_System_Threading",
+#    "Win32_System_WindowsProgramming",
+#]
