diff --git a/Cargo.toml b/Cargo.toml
index 076f32f..67bf8af 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,17 +50,17 @@ default = []
 macros = ["dep:deranged-macros"]
 num = ["dep:num-traits"]
 powerfmt = ["dep:powerfmt"]
-quickcheck = [
-    "dep:quickcheck",
-    "alloc",
-]
+#quickcheck = [
+#    "dep:quickcheck",
+#    "alloc",
+#]
 rand = [
-    "rand08",
+#    "rand08",
     "rand09",
-    "rand010",
+#    "rand010",
 ]
-rand010 = ["dep:rand010"]
-rand08 = ["dep:rand08"]
+#rand010 = ["dep:rand010"]
+#rand08 = ["dep:rand08"]
 rand09 = ["dep:rand09"]
 serde = ["dep:serde_core"]

@@ -82,22 +82,22 @@ version = "0.2.0"
 optional = true
 default-features = false

-[dependencies.quickcheck]
-version = "1.0.3"
-optional = true
-default-features = false
-
-[dependencies.rand010]
-version = "0.10.0"
-optional = true
-default-features = false
-package = "rand"
-
-[dependencies.rand08]
-version = "0.8.4"
-optional = true
-default-features = false
-package = "rand"
+#[dependencies.quickcheck]
+#version = "1.0.3"
+#optional = true
+#default-features = false
+
+#[dependencies.rand010]
+#version = "0.10.0"
+#optional = true
+#default-features = false
+#package = "rand"
+
+#[dependencies.rand08]
+#version = "0.8.4"
+#optional = true
+#default-features = false
+#package = "rand"

 [dependencies.rand09]
 version = "0.9.0"
