diff --git a/Cargo.toml b/Cargo.toml
index 9a7ff62..ca8133b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -70,7 +70,7 @@ io-util = [
 ]
 join-map = [
     "rt",
-    "hashbrown",
+#    "hashbrown",
 ]
 net = ["tokio/net"]
 rt = [
@@ -216,10 +216,10 @@ version = "0.3.0"
 version = "0.3.0"
 optional = true

-[dependencies.hashbrown]
-version = "0.15.0"
-optional = true
-default-features = false
+#[dependencies.hashbrown]
+#version = "0.15.0"
+#optional = true
+#default-features = false

 [dependencies.pin-project-lite]
 version = "0.2.11"
