--- phf-generator.orig/Cargo.toml
+++ phf-generator/Cargo.toml
@@ -22,18 +22,6 @@
 repository = "https://github.com/rust-phf/rust-phf"
 resolver = "1"
 
-[[bin]]
-name = "gen_hash_test"
-required-features = ["criterion"]
-
-[[bench]]
-name = "benches"
-harness = false
-
-[dependencies.criterion]
-version = "0.3.6"
-optional = true
-
 [dependencies.phf_shared]
 version = "^0.11.2"
 default-features = false
@@ -42,6 +30,3 @@
 version = "0.8"
 features = ["small_rng"]
 default-features = false
-
-[dev-dependencies.criterion]
-version = "0.3.6"
