diff --git a/Cargo.toml b/Cargo.toml
index 03c5fee..559fb99 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -82,11 +82,11 @@ version = "1.4.2"
 [target."cfg(windows)".dependencies.encode_unicode]
 version = "1"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_UI_Input_KeyboardAndMouse",
-]
+#[target."cfg(windows)".dependencies.windows-sys]
+#version = "0.61"
+#features = [
+#    "Win32_Foundation",
+#    "Win32_System_Console",
+#    "Win32_Storage_FileSystem",
+#    "Win32_UI_Input_KeyboardAndMouse",
+#]
