diff --git a/Cargo.toml b/Cargo.toml
index cd7c4bc..22f627e 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,17 +39,17 @@ version = "0.5.0"
 [dependencies.hex]
 version = "0.4"

-[dependencies.kanidm-hsm-crypto]
-version = "^0.2.0"
-optional = true
+#[dependencies.kanidm-hsm-crypto]
+#version = "^0.2.0"
+#optional = true

-[dependencies.openssl]
-version = "^0.10.38"
-optional = true
+#[dependencies.openssl]
+#version = "^0.10.38"
+#optional = true

-[dependencies.openssl-kdf]
-version = "0.4.2"
-optional = true
+#[dependencies.openssl-kdf]
+#version = "0.4.2"
+#optional = true

 [dependencies.serde]
 version = "^1.0.136"
@@ -73,11 +73,11 @@ features = ["serde"]
 version = "^0.3.11"

 [features]
-default = ["openssl"]
-hsm-crypto = ["kanidm-hsm-crypto"]
-msextensions = [
-    "dep:openssl-kdf",
-    "openssl",
-    "kanidm-hsm-crypto/msextensions",
-]
+default = []
+#hsm-crypto = ["kanidm-hsm-crypto"]
+#msextensions = [
+#    "dep:openssl-kdf",
+#    "openssl",
+#    "kanidm-hsm-crypto/msextensions",
+#]
 unsafe_release_without_verify = []
