Index: compact-jwt/Cargo.toml
===================================================================
--- compact-jwt.orig/Cargo.toml
+++ compact-jwt/Cargo.toml
@@ -39,10 +39,6 @@ version = "0.5.0"
 [dependencies.hex]
 version = "0.4"
 
-[dependencies.kanidm-hsm-crypto]
-version = "^0.2.0"
-optional = true
-
 [dependencies.openssl]
 version = "^0.10.38"
 optional = true
@@ -74,10 +70,8 @@ version = "^0.3.11"
 
 [features]
 default = ["openssl"]
-hsm-crypto = ["kanidm-hsm-crypto"]
 msextensions = [
     "dep:openssl-kdf",
     "openssl",
-    "kanidm-hsm-crypto/msextensions",
 ]
 unsafe_release_without_verify = []
