diff --git a/Cargo.toml b/Cargo.toml
index 54a9678..a8efac8 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,17 +46,11 @@ status = "actively-developed"
 [features]
 default = ["std"]
 libm = ["num-traits/libm"]
-partial_fixed_point_support = [
-    "fixed",
-    "cordic",
-]
 portable_simd = ["std"]
-rkyv-serialize = ["rkyv"]
 serde_serialize = [
     "serde",
-    "fixed/serde",
 ]
-std = ["wide/std"]
+std = []

 [lib]
 name = "simba"
@@ -66,16 +60,16 @@ path = "src/lib.rs"
 version = "0.5"
 default-features = false

-[dependencies.cordic]
+[disabled.dependencies.cordic]
 version = "0.1"
 optional = true

-[dependencies.decimal]
+[disabled.dependencies.decimal]
 version = "2.0"
 optional = true
 default-features = false

-[dependencies.fixed]
+[disabled.dependencies.fixed]
 version = "1"
 optional = true

@@ -95,11 +89,11 @@ default-features = false
 [dependencies.paste]
 version = "1.0"

-[dependencies.rand]
+[disabled.dependencies.rand]
 version = "0.8"
 optional = true

-[dependencies.rkyv]
+[disabled.dependencies.rkyv]
 version = "0.7"
 optional = true

@@ -108,7 +102,7 @@ version = "1"
 optional = true
 default-features = false

-[dependencies.wide]
+[disabled.dependencies.wide]
 version = "0.7"
 optional = true
 default-features = false
