diff --git a/Cargo.toml b/Cargo.toml
index bedf0a4..935ea4b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,18 +50,14 @@ alloc = [
 ]
 core-num-saturating = []
 crc = ["dep:crc"]
-default = ["heapless-cas"]
+default = []
 defmt = ["dep:defmt"]
 embedded-io = ["dep:embedded-io-04"]
 embedded-io-04 = ["dep:embedded-io-04"]
 embedded-io-06 = ["dep:embedded-io-06"]
 experimental-derive = ["postcard-derive"]
-heapless = ["dep:heapless"]
-heapless-cas = [
-    "heapless",
-    "dep:heapless",
-    "heapless/cas",
-]
+heapless = []
+heapless-cas = []
 nalgebra-v0_33 = ["nalgebra_v0_33"]
 paste = []
 postcard-derive = ["dep:postcard-derive"]
@@ -114,7 +110,7 @@ version = "0.6"
 optional = true
 package = "embedded-io"

-[dependencies.heapless]
+[disable.dependencies.heapless]
 version = "0.7.0"
 features = ["serde"]
 optional = true
