diff --git a/Cargo.toml b/Cargo.toml
index 7b4d9f0..8d2d9ef 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -49,15 +49,13 @@ rustdoc-args = ["--generate-link-to-definition"]
 all-features = true

 [features]
-os_rng = ["dep:getrandom"]
 serde = ["dep:serde"]
-std = ["getrandom?/std"]

 [lib]
 name = "rand_core"
 path = "src/lib.rs"

-[dependencies.getrandom]
+[disabled.dependencies.getrandom]
 version = "0.3.0"
 optional = true

