Index: openidconnect/Cargo.toml
===================================================================
--- openidconnect.orig/Cargo.toml
+++ openidconnect/Cargo.toml
@@ -43,7 +43,7 @@ name = "okta_device_grant"
 required-features = ["reqwest-blocking"]
 
 [dependencies.base64]
-version = "0.21"
+version = "0.22"
 
 [dependencies.chrono]
 version = "0.4"
@@ -68,7 +68,7 @@ version = "0.12.1"
 version = "1.0"
 
 [dependencies.itertools]
-version = "0.10"
+version = "0.13"
 
 [dependencies.log]
 version = "0.4"
