diff --git a/Cargo.toml b/Cargo.toml
index 3361c27..b7a8976 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@

 [package]
 edition = "2024"
-rust-version = "1.88"
+rust-version = "1.85"
 name = "home"
 version = "0.5.12"
 authors = ["Brian Anderson <andersrb@gmail.com>"]
@@ -40,13 +40,13 @@ resolver = "2"
 name = "home"
 path = "src/lib.rs"

-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_System_Com",
-]
+#[target."cfg(windows)".dependencies.windows-sys]
+#version = "0.61"
+#features = [
+#    "Win32_Foundation",
+#    "Win32_UI_Shell",
+#    "Win32_System_Com",
+#]

 [lints.clippy]
 dbg_macro = "warn"
